sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -710,6 +710,7 @@ public <T> CosmosPagedFlux<T> queryItems(String query, Class<T> classType) {
      *
      *  @return Mono of Void.
      */
+    @Deprecated
```

Comment:
Please add link to the new PAI to-be-used instead

Reply:
added